openapi/paths/payout-requests.yaml

```
@@ -40,7 +40,10 @@ post:
   summary: Create a payout request
   operationId: PostPayoutRequest
   x-sdk-operation-name: create
-  description: Creates a payout request.
+  description: |-
+    Creates a payout request.
+    `selectPaymentInstrumentUrl` from the response is used to redirect the customer to select a preferred payment instrument.
+    After payment instrument is selected, the customer is redirected to the `selectedPaymentInstrumentRedirectUrl`.
```

nitpick: where possible, avoid starting with a variable name (this helps readability)
```suggestion
    In the response, the `selectPaymentInstrumentUrl` field is used to redirect the customer to select a preferred payment instrument.
    After a payment instrument is selected, the customer is redirected to the `selectedPaymentInstrumentRedirectUrl` value.
```